Meta/ladybird.py

```diff
@@ -286,6 +295,39 @@ def _build_main(build_dir, **kwargs):
         sys.exit(1)
 
 
+def _test_main(build_dir, **kwargs):
+    build_preset = kwargs.get('preset')
+    test_args = [
+        'ctest',
+        '--preset',
+        build_preset,
+        '--output-on-failure',
+        '--test-dir',
+        str(build_dir),
+    ]
+    test_name_pattern = kwargs.get('pattern', None)
+    if test_name_pattern:
+        if test_name_pattern == 'WPT':
+            test_args.extend([
+                '-C',
+                'Integration',
+            ])
```

Comment:
I think we removed the WPT test in the Integration component a while ago. you have to run WPT.sh instead of using the main build dir's ctest tests.